supervisor/homeassistant/websocket.py

```diff
@@ -22,61 +26,112 @@ class WSClient:
     """Home Assistant Websocket client."""
 
     def __init__(
-        self, ha_version: AwesomeVersion, client: aiohttp.ClientWebSocketResponse
+        self,
+        loop: asyncio.BaseEventLoop,
+        ha_version: AwesomeVersion,
+        client: aiohttp.ClientWebSocketResponse,
     ):
         """Initialise the WS client."""
-        self.ha_version: AwesomeVersion = ha_version
-        self.client: aiohttp.ClientWebSocketResponse = client
-        self.message_id: int = 0
-        self._lock: asyncio.Lock = asyncio.Lock()
+        self.ha_version = ha_version
+        self._client = client
+        self._message_id: int = 0
```

Comment:
I think that gets messed up on the restart of the supervisor during the runtime? Maybe would be better to use https://github.com/home-assistant/core/blob/dev/homeassistant/util/uuid.py#L12

Reply:
I have restarted core and SU multiple times with this, there were no issues detected so far

Reply:
Home Assistant also expect an int for "id" and does not allow for any additional fields that we could use for uuid